workloads/vars/fio.yml

```diff
@@ -0,0 +1,63 @@
+
+# Use nodeselector to place workload job on specific node
+workload_job_node_selector: "{{ lookup('env', 'WORKLOAD_JOB_NODE_SELECTOR')|default('', true)|bool }}"
+# Tolerate taint on workload driver
+workload_job_taint: "{{ lookup('env', 'WORKLOAD_JOB_TAINT')|default(true, true)|bool }}"
+# Privileged job container
+workload_job_privileged: "{{ lookup('env', 'WORKLOAD_JOB_PRIVILEGED')|default(true, true)|bool }}"
+
+# Kubeconfig for tooling container script
+kubeconfig_file: "{{ lookup('env', 'KUBECONFIG_FILE')|default('~/.kube/config', true) }}"
+
+# pbench variables
+pbench_ssh_private_key_file: "{{ lookup('env', 'PBENCH_SSH_PRIVATE_KEY_FILE')|default('~/.ssh/id_rsa', true) }}"
+pbench_ssh_public_key_file: "{{ lookup('env', 'PBENCH_SSH_PUBLIC_KEY_FILE')|default('~/.ssh/id_rsa.pub', true) }}"
+enable_pbench_agents: "{{ lookup('env', 'ENABLE_PBENCH_AGENTS')|default(false, true)|bool }}"
+pbench_server: "{{ lookup('env', 'PBENCH_SERVER')|default('', true) }}"
+
+# Other variables for workload tests
+scale_ci_results_token: "{{ lookup('env', 'SCALE_CI_RESULTS_TOKEN')|default('', true) }}"
+job_completion_poll_attempts: "{{ lookup('env', 'JOB_COMPLETION_POLL_ATTEMPTS')|default(360, true)|int }}"
+
+# FIO IO test workload specific parameters:
+fiotest_prefix: "{{ lookup('env', 'FIOTEST_PREFIX')|default('fiotest', true) }}"
+fiotest_cleanup: "{{ lookup('env', 'FIOTEST_CLEANUP')|default(true, true)|bool|lower }}"
+fiotest_basename: "{{ lookup('env', 'FIOTEST_BASENAME')|default('fiotest', true) }}"
+fiotest_maxpods: "{{ lookup('env', 'FIOTEST_MAXPODS')|default(6, true)|int }}"
+fiotest_pod_image: "{{ lookup('env', 'FIOTEST_POD_IMAGE')|default('quay.io/openshift-scale/scale-ci-fio', true) }}"
+fiotest_stepsize: "{{ lookup('env', 'FIOTEST_STEPSIZE')|default(1, true)|int }}"
+fiotest_pause: "{{ lookup('env', 'FIOTEST_PAUSE')|default(0, true)|int }}"
+fiotest_storage_size: "{{ lookup('env', 'FIOTEST_STORAGE_SIZE')|default('1Gi',true) }}"
+fiotest_storageclass: "{{ lookup('env', 'FIOTEST_STORAGECLASS')|default('csi-rbd', true) }}"
+access_modes: "{{ lookup('env', 'ACCESS_MODES')|default('ReadWriteOnce',true) }}"
+fiotest_bs: "{{ lookup('env', 'FIOTEST_BS')|default(4,true)|int }}"
+fiotest_filename: "{{ lookup('env', 'FIOTEST_FILENAME')|default('/mnt/pvcmount/f2', true) }}"
+fiotest_filesize: "{{ lookup('env', 'FIOTEST_FILESIZE')|default('1G', true) }}"
+fiotest_runtime: "{{ lookup('env', 'FIOTEST_RUNTIME')|default(60,true)|int }}"
+fiotest_direct: "{{ lookup('env', 'FIOTEST_DIRECT')|default(1, true)|int }}" 
+fiotest_iodepth: "{{ lookup('env', 'FIOTEST_IODEPTH')|default(32,true)|int }}" 
+fiotest_testtype: "{{ lookup('env', 'FIOTEST_TESTTYPE')|default('write',true) }}" 
+fiotest_samples: "{{ lookup('env', 'FIOTEST_SAMPLES')|default(1,true)|int }}"
+storage_fio_test_image: "{{ lookup('env', 'STORAGE_FIO_TEST_IMAGE')|default('quay.io/openshift-scale/scale-ci-fio', true) }}"
```

Comment:
It would be nice to have these and env vars consistent.  Again, change to all the variables from `storage_fio_test` to `fiotest_storage`?  Also, the environment variables too (`s/STORAGE_FIO_TEST_/FIOTEST_STORAGE_/`).